custom_components/tuya_local/devices/poolsana_heatpump.yaml

```diff
@@ -0,0 +1,85 @@
+name: Pool heat pump
+primary_entity:
+  products:
+  - id: abb8kaxcphfzkwsg
+    name: PoolSana InverPower Next
+  entity: climate
+  translation_key: pool_heatpump
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cooling
+              value: cool
+            - dps_val: Heating
+              value: heat
+            - dps_val: Auto
+              value: auto
+    - id: 2
+      type: string
+      name: mode
+      hidden: true
+    - id: 4
+      type: integer
+      name: temperature
+      range:
+        min: 70
+        max: 400
+      mapping:
+        - scale: 10
+          step: 10
+    - id: 16
+      type: integer
+      name: current_temperature
+      range:
```

Comment:
Range is not needed here, as current_temperature only reports what the device is saying.  Step is also not needed (for that reason, and also that a step of 1 is the default)